### ISSUE TITLE
Importing data from MS SQL to MongoDB 3.4 

### ISSUE THREAD

**Issue by SandraArnold** [msathis/SQLToNoSQLImporter#12]
We are thinking about using your tool to import data from a MS SQL database to MongoDB 3.4 on RedHat 7.  Can the tool be installed on the RedHat server or does it need to run on a Windows server in order to use the MS SQL driver?

Thanks

Sandra Arnold
Senior DBA
TiVo

**Comment by msathis** [comment]
Yes. It should work anywhere as its written in java. Please let me know if you face any issues :)
